config/default.yml

```diff
@@ -620,33 +620,33 @@ RSpec/Yield:
   VersionAdded: '1.32'
   StyleGuide: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/Yield
 
-Capybara/CurrentPathExpectation:
+RSpec/Capybara/CurrentPathExpectation:
   Description: Checks that no expectations are set on Capybara's `current_path`.
   Enabled: true
   VersionAdded: '1.18'
```

Comment:
Does this still apply when the cop is renamed?

Reply:
Let me quickly check how RuboCop itself does it.

Reply:
It seems that it's treated as a new cop, `VersionAdded` is set to the release version, and `VersionChanged` is reset. At least judging by [this example](https://github.com/rubocop-hq/rubocop/commit/f5b9401c4a5354b72100a8c1ffa33076942eeadc#diff-3653213e3a153018a75cc7b2536b1d0217a6d8d75188d71cf45441812f789698R3408).

Reply:
Interesting. Another example is https://github.com/rubocop-hq/rubocop/commit/8ccdf3e1980b6349a841196f00329c12c6e6bfa1 where `VersionChanged` is updated, but `VersionAdded` is unchanged.

Reply:
Also see config/default.yml in https://github.com/rubocop-hq/rubocop/pull/7468/files – only `VersionChanged` is updated.

Reply:
I’ll go with changing `VersionChanged`.